### ISSUE TITLE
Problems with clusterDomain and operator

### ISSUE THREAD

**Issue by Lamercho** [cockroachdb/cockroach-operator#893]
What is the proper way to install cockroachdb with operator and set custom cluster domain different than cluster.local ?

In helm chart there was a config variable for that, cant seem to find one for the operator config.

**Comment by meridional** [comment]
Hey @Lamercho. Sorry for the delay. I think this PR (#894) that I put up might not be the best way to support custom domains. In particular, it will not work well in some of the custom DNS setups.

Are you running into issues with the operator in a k8s cluster with a custom cluster domain? If your clients connect to the cluster without using the fully qualified domain name (`crdb.some-namespace.svc` instead of `crdb.some-namespace.svc.<your-domain`), it should work.

**Comment by Lamercho** [comment]
> Hey @Lamercho. Sorry for the delay. I think this PR (#894) that I put up might not be the best way to support custom domains. In particular, it will not work well in some of the custom DNS setups.
> 
> Are you running into issues with the operator in a k8s cluster with a custom cluster domain? If your clients connect to the cluster without using the fully qualified domain name (`crdb.some-namespace.svc` instead of `crdb.some-namespace.svc.<your-domain`), it should work.

Yep we are doing it without FQDN but it will be great to have something like --cluster-dmain parameter during crdb installation.
